app/javascript/src/components/Contacts/Create/index.jsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Pane, Typography, Button } from "@bigbinary/neetoui/v2";
+import { Input, Select } from "@bigbinary/neetoui/v2/formik";
+import { Formik, Form } from "formik";
+import PropTypes from "prop-types";
+
+import formInitialValue from "constants/formInitialValues";
+import formValidationSchema from "constants/formValidationSchemas";
```

Comment:
Please move `NOTES_INITIAL_VALUES` and `NOTE_VALIDATION_SCHEMA` to a constants file in the scope of this directory.